README.md

```diff
@@ -18,6 +18,84 @@ To get updates, watch the repo and follow the [Twitter account](https://twitter
 
 # Tips list
 
+## #1(number) - Alternatives to classic `for` loops
+
+> yyyy/mm/dd(date) by [@chtefid](https://twitter.com/chtefid)
+
+Everybody knows the classic `for` loop :
+
+```javascript
+for (var i = 0; i < elements.length; i++) {
+  var element = elements[i];
+  doSomethingWith(element);
+}
+```
+
+But often, you don't need `i` anywhere else except to get the element at this index.
+You just want to loop over your elements to do something with them.
+An alternative is to use the ES5 Array functions, such as `forEach` :
```

Comment:
Better document an important problem by the functional `forEach` which is that you can't break the loop. The alternative for it is to use `every` instead:

```javascript
var BREAK = false;
var CONTINUE = true;
array.every(function( item ) {
  if ( item.shouldBreak ) {
    return BREAK;
  }
  return CONTINUE;
});
```